python/tests/alarm_clock/alarm_clock.yaml

```
@@ -1,6 +1,7 @@
 import:
   files:
     - ./structures.yaml
+    - ../alarm_clock/structures.yaml
```

```suggestion
    - ./structures.yaml
```

This is the same import, don't need both. I had removed it in mine.

---

Must have happened in the merge, removed.